docs/resources/http.md.erb

```diff
@@ -19,10 +19,9 @@ Executing the HTTP test on the remote target will be the default behavior in InS
 
 An `http` resource block declares the configuration settings to be tested:
 
-    describe http('url', auth: {user: 'user', pass: 'test'}, params: {params}, method: 'method', headers: {headers}, data: data, open_timeout: 60, read_timeout: 60, ssl_verify: true) do
-      its('status') { should eq number }
-      its('body') { should eq 'body' }
-      its('headers.name') { should eq 'header' }
+    describe http('http://www.example.com') do
+      its('body') { should cmp 'awesome' }
+      its('status') { should cmp 200 }
     end
 
 where
```

Comment:
I agree that a useful test is better...but how should we explain the filter criteria?

Reply:
A bit further down we give a "complex example"

https://github.com/chef/inspec/blob/285c9b5077c437754647a07b30ce6cec89e2ec70/docs/resources/http.md.erb#L62

Will that work?